.github/workflows/continous-integration-docker.yml

```diff
@@ -522,6 +521,14 @@ jobs:
       root_state_path: '/var/cache'
       zebra_state_dir: 'zebrad-cache'
       lwd_state_dir: 'lwd-cache'
+    # We want to prevent multiple lightwalletd send transaction tests running at the same time,
```

Comment:
I added the standard concurrency rule to the PR, like PR #5393.

We might need this rule because the test runs for a long time (2-3 hours, or longer if the cached state is old).

Reply:
Thank you!